metadata.json

```diff
@@ -48,6 +48,13 @@
         "9"
       ]
     },
+    {
+      "operatingsystem": "Amazon",
+      "operatingsystemrelease": [
+        "2.0",
```

Comment:
There are both `Amazon` and `AmazonLinux`, and `2.0` and `2` 🤔: https://github.com/search?q=org%3Avoxpupuli+path%3Ametadata.json+amazon&type=code